website/www/site/content/en/get-started/downloads.md

```diff
@@ -96,10 +96,17 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.56.0 (2024-05-01)
+Official [source code download](https://downloads.apache.org/beam/2.56.0/apache-beam-2.56.0-source-release.zip).
```

Comment:
they are not available yet. Checked that the url is correct. Maybe just a delay?

Reply:
I think its because PMC finalization hasn't happened yet